packages/eas-cli/src/commands/build/run.ts

```diff
@@ -38,8 +41,6 @@ interface RunCommandFlags {
 }
 
 export default class Run extends EasCommand {
-  static override hidden = true;
-
```

Comment:
I think that we can make the `build:run` command publicly available now 🎉 

CC: @brentvatne